README.md

```diff
@@ -115,16 +115,111 @@ python -m pytest 'tests/' -v
 
 Ensure you have your environment configured correctly (as described above).
 
-### Google Cloud Function
+### Google Cloud Functions
+
+iCone, COTrip, and NavJoy 568 message translators have been hosted in the GCP as cloud functions. All functions will publish generated messages to the wzdx_messages pub/sub topic in their respective environments, and will publish unsupported/invalid messages to a sepate topic, specified in the environment variables of each function. 
+
+#### iCone
+The iCone cloud function is triggered once per day by a pub/sub topic. The cloud function is located at [https://console.cloud.google.com/functions/details/us-central1/auto_icone_translator_ftp?project=cdot-oim-wzdx-dev](https://console.cloud.google.com/functions/details/us-central1/auto_icone_translator_ftp?project=cdot-oim-wzdx-dev)
 
 A system was created in google cloud platform to automatically translate iCone data to WZDx message. This system consists of two pubsub topics and a cloud function. A cloud scheduler automatically sends a message to a pubsub topic which triggers the cloud function. The cloud function retrieves iCone data from an ftp server (ftp://iconetraffic.com:42663) and translates to WZDx message. It validates the WZDx message with json schema and publishes the message to a pubsub topic.
 
 ![alt text](translator/GCP_cloud_function/iCone%20Translator%20block%20diagram.png)
```

Comment:
This diagram seems to have been moved, but we include it below properly. Do we just want to remove the reference here?